app/templates/validation-reports/details.hbs

```diff
@@ -6,4 +6,56 @@
   </strong>
 </BreadcrumbsItem>
 
-{{outlet}}
\ No newline at end of file
+<AuDataTable
+  @content={{this.model.results}}
+  @fields="FocusNode ResultMessage ResultPath SourceConstraint Value"
```

Comment:
`@fields` isn't required if you specify the needed columns in the content block.